test/ufuzz.js

```diff
@@ -192,11 +187,12 @@ var ASSIGNMENTS = [
     '=',
     '=',
     '=',
+    '=',
+    '=',
+    '=',
+    '=',
+    '=',
 
-    '==',
-    '!=',
-    '===',
-    '!==',
     '+=',
```

Comment:
Adding a couple more `'+='` entries to ASSIGNMENTS might be a good idea as it's also string friendly and might lead to interesting results.